library/core/src/num/mod.rs

```diff
@@ -993,10 +995,7 @@ from_str_radix_int_impl! { isize i8 i16 i32 i64 i128 usize u8 u16 u32 u64 u128 }
 
 macro_rules! doit {
```

Comment:
I know this isn't your problem, but while you're here could you rename `doit` to something more helpful?  Perhaps `impl_helper_for` or something.

Reply:
sure.